source/contracts/Controller.sol

```diff
@@ -98,32 +90,18 @@ contract Controller is IController {
         return registry[_key].contractAddress;
     }
 
-    function suicideFunds(IControlled _target, address _destination, ERC20Basic[] _tokens) public devModeOwnerOnly returns (bool) {
-        _target.suicideFunds(_destination, _tokens);
-        return true;
-    }
-
-    function updateController(IControlled _target, Controller _newController) public devModeOwnerOnly returns (bool) {
-        _target.setController(_newController);
+    function transferOwnership(address _newOwner) public onlyOwnerCaller returns (bool) {
+        owner = _newOwner;
         return true;
     }
 
-     /*
-     * Controller Administration [dev can transfer ownership anytime, mode can only switched from dev mode -> decentralized]
-     */
-
-    function transferOwnership(address _newOwner) public onlyOwnerCaller returns (bool) {
-        // if in dev mode, blacklist old owner and whitelist new owner
-        if (whitelist[owner]) {
-            whitelist[owner] = false;
-            whitelist[_newOwner] = true;
-        }
-        owner = _newOwner;
+    function emergencyStop() public onlyOwnerCaller onlyInGoodTimes returns (bool) {
+        stopped = true;
         return true;
     }
 
-    function switchOffDevMode() public devModeOwnerOnly returns (bool) {
-        whitelist[owner] = false;
+    function release() public onlyOwnerCaller onlyInBadTimes returns (bool) {
```

Comment:
Not a huge fan of this name, though I don't have any great recommendations.  Would be nice to come up with a name that is somehow a mirror of `emergencyStop`.  Perhaps just renaming both to `start` and `stop`?  Or `emergencyStop` and `emergencyStart`?  I recognize the problems with both of these, but at least they are more clearly paired.

Reply:
I'm super ambivalent about this point (despite generally agreeing) especially since it will likely only ever be used by me in development and shouldn't ever be called outside of that context.